manifest.toml

```diff
@@ -7,7 +7,7 @@ description.fr = "Système de gestion de contenu basé sur Wiki"
 
 version = "26.2~ynh1"
 
-maintainers = ["eric_G"]
```

Comment:
I am a community member of Tiki Wiki CMS Groupware.

Is there anything I should know about? I can be reached in public or private.

Thanks!

Reply:
Just that @ericgaspar wants to reduce the amount of pressure and pings he can receive (not from tiki especially, but in general). We are not dropping the package, don't worry.

Reply:
I appreciate everything @ericgaspar and I thank you @Salamandar for stepping up.

Reply:
I'm not stepping up to become this package's maintainer, but we are still trying to maintain it (at least update it / fix when bugs). :)

Reply:
@Salamandar: Understood. @rodriguezny can maintain it. He is a good coder, but new to YunoHost. Can you guide him to learn?

Thanks!